Failed tests:
  CircuitBreakerBulkheadTest>Arquillian.run:138->testCircuitBreakerAroundBulkheadAsync:126 Unexpected exception thrown from Future
  CircuitBreakerBulkheadTest>Arquillian.run:138->testCircuitBreakerAroundBulkheadSync:181 Unexpected exception thrown from Future
  TimeoutUninterruptableTest>Arquillian.run:138->testTimeoutAsyncBulkhead:185 Execution count after second call expected [1] but found [2]
  TimeoutUninterruptableTest>Arquillian.run:138->testTimeoutAsyncBulkheadQueueTimed:237 Time taken for call B to timeout
Expected: a value less than <PT0.75S>
     but: <PT0.810478211S> was greater than <PT0.75S>
  BulkheadSynchConfigTest>Arquillian.run:138->testBulkheadClassSemaphore3:83  Work is not being done simultaneously enough, only 3  workers at once. Expecting 5.  expected [true] but found [false]
  BulkheadSynchTest>Arquillian.run:138->testBulkheadExceptionThrownWhenQueueFullSemaphore:199 Unable to clean up all tasks
  FallbackMethodOutOfPackageTest>Arquillian.arquillianBeforeClass:96 ? Runtime E...
  FallbackMethodSubclassTest>Arquillian.arquillianBeforeClass:96 ? Runtime Expec...
  FallbackMethodSuperclassPrivateTest>Arquillian.arquillianBeforeClass:96 ? Runtime
  FallbackMethodWildcardNegativeTest>Arquillian.arquillianBeforeClass:96 ? Runtime
  IncompatibleFallbackMethodTest>Arquillian.arquillianBeforeClass:96 ? Runtime E...
  IncompatibleFallbackMethodWithArgsTest>Arquillian.arquillianBeforeClass:96 ? Runtime
  IncompatibleFallbackPolicies>Arquillian.arquillianBeforeClass:96 ? Runtime Exp...
  IncompatibleFallbackTest>Arquillian.arquillianBeforeClass:96 ? Runtime Expecte...
  InvalidAsynchronousClassTest>Arquillian.arquillianBeforeClass:96 ? Runtime Exp...
  InvalidAsynchronousMethodTest>Arquillian.arquillianBeforeClass:96 ? Runtime Ex...
  InvalidBulkheadAsynchQueueTest>Arquillian.arquillianBeforeClass:96 ? Runtime E...
  InvalidBulkheadValueTest>Arquillian.arquillianBeforeClass:96 ? Runtime Expecte...
  InvalidCircuitBreakerDelayTest>Arquillian.arquillianBeforeClass:96 ? Runtime E...
  InvalidCircuitBreakerFailureRatioNegTest>Arquillian.arquillianBeforeClass:96 ? Runtime
  InvalidCircuitBreakerFailureRatioPosTest>Arquillian.arquillianBeforeClass:96 ? Runtime
  InvalidCircuitBreakerFailureReqVol0Test>Arquillian.arquillianBeforeClass:96 ? Runtime
  InvalidCircuitBreakerFailureReqVolNegTest>Arquillian.arquillianBeforeClass:96 ? Runtime
  InvalidCircuitBreakerFailureSuccess0Test>Arquillian.arquillianBeforeClass:96 ? Runtime
  InvalidCircuitBreakerFailureSuccessNegTest>Arquillian.arquillianBeforeClass:96 ? Runtime
  InvalidRetryDelayDurationTest>Arquillian.arquillianBeforeClass:96 ? Runtime Ex...
  InvalidRetryDelayTest>Arquillian.arquillianBeforeClass:96 ? Runtime Expected e...
  InvalidRetryJitterTest>Arquillian.arquillianBeforeClass:96 ? Runtime Expected ...
  InvalidRetryMaxRetriesTest>Arquillian.arquillianBeforeClass:96 ? Runtime Expec...
  InvalidTimeoutValueTest>Arquillian.arquillianBeforeClass:96 ? Runtime Expected...
  BulkheadMetricTest>Arquillian.run:138->bulkheadMetricAsyncTest:244 queue wait histogram counts
Expected: is <4L>
     but: was <5L>
  BulkheadMetricTest>Arquillian.run:138->bulkheadMetricHistogramTest:184 histogram count
Expected: is <2L>
     but: was <3L>
  BulkheadMetricTest>Arquillian.run:138->bulkheadMetricTest:122 bulkhead wait time histogram present
Expected: is <false>
     but: was <true>
  CircuitBreakerMetricTest>Arquillian.run:138->testCircuitBreakerMetric:136 circuitbreaker calls prevented
Expected: is <1L>
     but: was <3L>
  FallbackMetricTest>Arquillian.run:138->fallbackMetricHandlerTest:93 failed invocations
Expected: is <0L>
     but: was <-1L>
  FallbackMetricTest>Arquillian.run:138->fallbackMetricMethodTest:67 failed invocations
Expected: is <0L>
     but: was <-1L>
  RetryVisibilityTest>Arquillian.run:138->serviceOverrideClassLevelUsesClassLevelAnnotation:134->checkServiceCall:248 in RetryVisibilityTest#serviceOverrideClassLevelUsesClassLevelAnnotation service() should have been called exactly 5 times expected [5] but found [4]

Tests run: 455, Failures: 37, Errors: 0, Skipped: 48

---

Failed tests:
  CircuitBreakerBulkheadTest>Arquillian.run:138->testCircuitBreakerAroundBulkheadAsync:126 Unexpected exception thrown from Future
  CircuitBreakerBulkheadTest>Arquillian.run:138->testCircuitBreakerAroundBulkheadSync:181 Unexpected exception thrown from Future
  TimeoutUninterruptableTest>Arquillian.run:138->testTimeoutAsyncBulkhead:185 Execution count after second call expected [1] but found [2]
  TimeoutUninterruptableTest>Arquillian.run:138->testTimeoutAsyncBulkheadQueueTimed:237 Time taken for call B to timeout
Expected: a value less than <PT0.75S>
     but: <PT0.809473006S> was greater than <PT0.75S>
  BulkheadSynchConfigTest>Arquillian.run:138->testBulkheadClassSemaphore3:83  Work is not being done simultaneously enough, only 3  workers at once. Expecting 5.  expected [true] but found [false]
  BulkheadSynchTest>Arquillian.run:138->testBulkheadExceptionThrownWhenQueueFullSemaphore:199 Unable to clean up all tasks
  BulkheadMetricTest>Arquillian.run:138->bulkheadMetricAsyncTest:244 queue wait histogram counts
Expected: is <4L>
     but: was <5L>
  BulkheadMetricTest>Arquillian.run:138->bulkheadMetricHistogramTest:184 histogram count
Expected: is <2L>
     but: was <3L>
  BulkheadMetricTest>Arquillian.run:138->bulkheadMetricTest:122 bulkhead wait time histogram present
Expected: is <false>
     but: was <true>


  CircuitBreakerMetricTest>Arquillian.run:138->testCircuitBreakerMetric:136 circuitbreaker calls prevented
Expected: is <1L>
     but: was <3L>
  FallbackMetricTest>Arquillian.run:138->fallbackMetricHandlerTest:93 failed invocations
Expected: is <0L>
     but: was <-1L>
  FallbackMetricTest>Arquillian.run:138->fallbackMetricMethodTest:67 failed invocations
Expected: is <0L>
     but: was <-1L>
  RetryVisibilityTest>Arquillian.run:138->serviceOverrideClassLevelUsesClassLevelAnnotation:134->checkServiceCall:248 in RetryVisibilityTest#serviceOverrideClassLevelUsesClassLevelAnnotation service() should have been called exactly 5 times expected [5] but found [4]

Tests run: 407, Failures: 13, Errors: 0, Skipped: 0

--

Results :

Failed tests:
  CircuitBreakerBulkheadTest>Arquillian.run:138->testCircuitBreakerAroundBulkheadAsync:126 Unexpected exception thrown from Future
  CircuitBreakerBulkheadTest>Arquillian.run:138->testCircuitBreakerAroundBulkheadSync:181 Unexpected exception thrown from Future
  TimeoutUninterruptableTest>Arquillian.run:138->testTimeoutAsyncBulkhead:181 Execution count after second call expected [1] but found [2]
  TimeoutUninterruptableTest>Arquillian.run:138->testTimeoutAsyncBulkheadQueueTimed:233 Time taken for call B to timeout
Expected: a value less than <PT0.75S>
     but: <PT0.810847813S> was greater than <PT0.75S>
  BulkheadSynchConfigTest>Arquillian.run:138->testBulkheadClassSemaphore3:83  Work is not being done simultaneously enough, only 3  workers at once. Expecting 5.  expected [true] but found [false]
  BulkheadSynchTest>Arquillian.run:138->testBulkheadExceptionThrownWhenQueueFullSemaphore:199 Unable to clean up all tasks
  RetryVisibilityTest>Arquillian.run:138->serviceOverrideClassLevelUsesClassLevelAnnotation:134->checkServiceCall:248 in RetryVisibilityTest#serviceOverrideClassLevelUsesClassLevelAnnotation service() should have been called exactly 5 times expected [5] but found [4]

Tests run: 407, Failures: 7, Errors: 0, Skipped: 0
